R/dp_deploy.R

```diff
@@ -22,30 +22,47 @@ dp_deploy <- function(project_path = ".", ...) {
   # get daap content and info
   conf <- dpconf_get(project_path = project_path)
   dlog <- get_dlog(project_path = project_path)
-  d <- readRDS(file = glue::glue(
-    "{project_path}/output_files/RDS_format/",
-    "data_object.RDS"
-  ))
+  # get the data object file type
+  type <- fs::dir_ls(file.path(project_path,'output_files/'), recurse = T, regexp = 'data_object') |> 
+      tools::file_ext() |> 
```

Comment:
We haven't used the native pipe anywhere else in the daapr codebase to my knowledge. Recommend keeping magrittr pipe for now for compatibility with older R versions